widgetssdk/src/main/java/com/glia/widgets/launcher/EngagementLauncher.kt

```diff
@@ -15,37 +16,66 @@ import com.glia.widgets.helper.TAG
  * audio calls, video calls, and secure messaging
  */
 interface EngagementLauncher {
+
+    /**
+     * Starts a chat engagement.
+     *
+     * @param context Activity or Context used to launch the chat screen
+     */
+    fun startChat(context: Context)
+
     /**
      * Starts a chat engagement.
      *
      * @param context Activity or Context used to launch the chat screen
-     * @param visitorContextAssetId Optional visitor context id from Glia Hub
+     * @param visitorContextAssetId Visitor context id from Glia Hub
```

Comment:
NIT. I remember one day tech writer advised to use visitor from a lowercase letter because it's not a name.

Reply:
Here and in other places below. 😉

Reply:
I'm wondering why the `Optional` was in uppercase

Reply:
`Activity` as well

Reply:
![image](https://github.com/user-attachments/assets/327cdc87-54da-4d5f-8a44-21e662bd5888)
Maybe this is some kind of beginning of a sentence?
So we should:
a) change both
b) keep both

Reply:
Ah, indeed, Optional should have been with lowercase because it's not a name as well. However, I don't remember regarding Activity. If I'm not mistaken, if it's used as an Android component or class name then it should be from the capital letter.

Reply:
In my opinion optional should start from lowercase (it's just a regular word) and Activity or Context from a capital letter (because these are the names of Android components) here. IMHO. Not critical, anyway.

Reply:
Hmm, I checked the `AppCompatActivity.java` class from Android library.
![image](https://github.com/user-attachments/assets/fd9b7195-2acf-4ea6-954c-721d83fb7c58)
It seems that the Word next to the `@param parameter-name` is considered the start of a sentence.

Reply:
Good to know, thank you. Sorry for confusion with `Visitor`.